umap/settings/base.py

```diff
@@ -274,7 +274,7 @@
 UMAP_MAPS_PER_SEARCH = 25
 UMAP_MAPS_PER_PAGE_OWNER = 10
 UMAP_SEARCH_CONFIGURATION = "simple"
-UMAP_HELP_URL = "https://wiki.openstreetmap.org/wiki/UMap#Feedback_and_help"
+UMAP_HELP_URL = "https://umap-project.org/"
```

Comment:
Why not https://discover.umap-project.org/ ?

Reply:
You're right 👍